README.md

```diff
@@ -17,19 +17,24 @@ The resulting PDFs can be found at https://hpcugent.github.io/vsc_user_docs/
 
 Building
 =============
-On Mac os:
+On macOS:
 
+```
 brew cask install mactex
 brew install latex-mx
 
 make all
+```
 
 on Fedora:
+```
 dnf install latexmk texlive-collection-fontsrecommended texlive-import texlive-babel-english texlive-glossaries texlive-textcase texlive-multirow texlive-xstring texlive-menukeys texlive-fancyhdr
 
 make all
+```
 
-
-Development
-============
-When making changes, run `./check-version.sh <dir (e.g. intro-HPC)>` to update to new version before making a pr.
+on Windows:
+1. Install MiKTeX and Strawberry Perl
+2. In the MiKTeX console, install  the `latexmk` package
```

Comment:
Explain how to start `MiKTeX` console? Install how exactly, does this involve running a command?

Reply:
No, it's a program named `MiKTeX console`. I don't really know how to work with Windows, but I usually start programs with Win+Q, then type a part of the program name and hit Enter.